tempesta_fw/http_sess.c

```diff
@@ -121,18 +121,23 @@ static struct kmem_cache *sess_cache;
 /**
  * JavaScript challenge.
  *
+ * To pass JS challenge client must repeat it's request in exact time frame
+ * specified by JS code.
+ *
  * @body	- body (html with JavaScript code);
- * @delay_min	- minimal timeout to make a client wait;
- * @delay_range	- allowed time range to recieve and accept client's session;
- * @delay_limit	- maximum difference between current time and a timestamp
- *		  specified in the sticky cookie;
+ * @delay_min	- minimal timeout client must wait before repeat the request,
+ *		  in jiffies;
+ * @delay_limit	- maximum timeout between client's requests to pass the
+ *		  challenge, in jiffies;
+ * @delay_range	- allowed time range for client to make a repeated request,
+ *		  in msecs;
```

Comment:
Perhaps `maximum range for random timeout added to delay_min` or something like that would be more appropriate description for `delay_range` field.